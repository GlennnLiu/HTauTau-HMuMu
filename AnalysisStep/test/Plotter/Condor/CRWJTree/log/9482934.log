000 (9482934.002.000) 07/27 15:05:28 Job submitted from host: <137.138.120.116:9618?addrs=137.138.120.116-9618+[2001-1458-d00-16--17b]-9618&alias=bigbird12.cern.ch&noUDP&sock=schedd_8624_ecd7>
...
040 (9482934.002.000) 07/27 15:06:57 Started transferring input files
	Transferring to host: <128.142.61.38:9618?addrs=128.142.61.38-9618+[2001-1458-301-23--100-25]-9618&alias=b7s06p8377.cern.ch&noUDP&sock=slot1_14_20102_6d6b_107527>
...
040 (9482934.002.000) 07/27 15:06:57 Finished transferring input files
...
001 (9482934.002.000) 07/27 15:06:58 Job executing on host: <128.142.61.38:9618?addrs=128.142.61.38-9618+[2001-1458-301-23--100-25]-9618&alias=b7s06p8377.cern.ch&noUDP&sock=startd_20065_7fe1>
...
006 (9482934.002.000) 07/27 15:07:07 Image size of job updated: 119276
	0  -  MemoryUsage of job (MB)
	0  -  ResidentSetSize of job (KB)
...
040 (9482934.002.000) 07/27 15:07:07 Started transferring output files
...
040 (9482934.002.000) 07/27 15:07:07 Finished transferring output files
...
005 (9482934.002.000) 07/27 15:07:07 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:00:04, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:04, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	23548  -  Run Bytes Sent By Job
	340  -  Run Bytes Received By Job
	23548  -  Total Bytes Sent By Job
	340  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1         1 
	   Disk (KB)            :       37        1   1310917 
	   Memory (MB)          :        0     2000      2000 

	Job terminated of its own accord at 2023-07-27T13:07:07Z.
...
