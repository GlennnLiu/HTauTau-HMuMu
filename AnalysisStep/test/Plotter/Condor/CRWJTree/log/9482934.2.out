[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRWJTree.diBoson.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRWJTree.triBoson.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRWJTree.top.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRWJTree.EWKZee.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRWJTree.EWKZtt.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRWJTree.EWKZmm.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRWJTree.DYee.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRWJTree.DYtt.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRWJTree.DYmm.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRWJTree.Wjet.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRWJTree.TTfake.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRWJTree.Data.root
[INFO] Total MC summed
[INFO] Saved Histos to /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRWJTree.TotalMC.root
